src/realm/sync/noinst/protocol_codec.hpp

```diff
@@ -164,9 +164,8 @@ class ClientProtocol {
 
     /// Messages sent by the client.
 
-    void make_bind_message(int protocol_version, OutputBuffer&, session_ident_type session_ident,
-                           const std::string& server_path, const std::string& signed_user_token,
-                           bool need_client_file_ident, bool is_subserver);
+    void make_bind_message(OutputBuffer&, session_ident_type session_ident, const std::string& path_data,
```

Comment:
I think we may need a FLX/PBS specific version of these functions like we have with IDENT messages to handle the different formats/protocol version handling here.

Reply:
Added the two different versions back and updated the tests.